packages/sources/spectral-macro-score/src/adapter.ts

```diff
@@ -0,0 +1,44 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { ExecuteFactory, AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { makeConfig, DEFAULT_ENDPOINT, SpectralAdapterConfig } from './config'
+import { MacroScoreAPI } from './endpoint'
+
+const inputParams = {
+  tokenIdInt: true,
+  tickSetId: true,
+}
+
+export const execute = async (
+  request: AdapterRequest,
+  config: SpectralAdapterConfig,
+): Promise<AdapterResponse> => {
```

Comment:
Here you should be able to make the constant of type `ExecuteWithConfig<SpectralAdapterConfig>`

Reply:
Should also make room for a context object to be passed in.

Reply:
Can you explain why I need a context here? Documentation would help.